docs/semgrep-appsec-platform/notifications.md

```diff
@@ -4,25 +4,71 @@ append_help_link: true
 title: Alerts and notifications
 hide_title: true
 toc_max_heading_level: 4
-description: "Learn how to receive Slack or email alerts about findings and failures and how to integrate using webhooks."
+description: "Learn how to receive Slack, webhook, or email notifications about findings."
 tags:
     - Deployment
     - Semgrep AppSec Platform
 ---
 
+import RuleModes from "/src/components/reference/_rule-modes.md"
+
 # Alerts and notifications
 
-Users can receive notifications from Semgrep AppSec Platform in the following channels:
+You can receive notifications for Semgrep Code and Supply Chain findings in the following channels:
```

Comment:
Alas, this isn't actually true. Only Slack works for SSC, the others only do code.

Reply:
Good callout. Updated the opening, removed the `ul` because the table served it better. It may have made the steps redundant, though  I also think keeping the general steps makes sure that people don't miss them (if they skim the table). WDYT?

Reply:
seems good, I left another comment about verifying specific behavior though 😅